frontend/public/locales/en/applications.json

```diff
@@ -20,6 +21,72 @@
+    "alt-phone-number": "Alternate phone number",
+    "email": "Email",
+    "mailing": "Mailing address",
+    "home": "Home address",
+    "comm-pref": "Communication preference",
+    "lang-pref": "Language preference",
+    "dental-private": "Access to dental insurance or coverage",
+    "dental-public": "Access to government dental benefits",
+    "application-code": "Application code",
+    "yes": "Yes",
+    "no": "No",
+    "exit": "Exit",
+    "close-application": "Close application",
+    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan.html",
+    "status-checker-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html",
+    "review-child-information": {
+      "read-carefully": "Please review this information carefully. The information provided will be used by Sun Life to create your child(ren)'s member account.",
+      "page-title": "Review child(ren) information",
+      "page-sub-title": "{{child}} information",
+      "dob-title": "Date of birth",
+      "dob-change": "Change date of birth",
+      "is-parent": "Parent or legal guardian of child",
+      "sin-title": "Social Insurance Number (SIN)",
+      "sin-change": "Change SIN",
+      "full-name-title": "Full name",
+      "full-name-change": "Change full name",
+      "dental-title": "{{child}} access to dental insurance",
+      "dental-insurance-title": "Access to other dental insurance or coverage",
+      "dental-insurance-change": "Change answer to dental insurance",
+      "dental-benefit-title": "Access to government dental benefits",
+      "dental-benefit-change": "Change answer to government dental benefits",
+      "dental-benefit-has-access": "Has access to the following benefits:",
+      "dental-benefit-has-no-access": "Has access to the following benefits:",
+      "submit-app-title": "Submit your application",
+      "submit-p-proceed": "By proceeding with this application, you are agreeing to attest truthfully to the information provided.",
+      "submit-p-false-info": "If you provide false information on your application, you and others you applied for may be removed from the plan.",
+      "submit-p-repayment": "If you or your family members were not eligible to apply, you or your family member may have to repay the full cost of care received through the Canadian Dental Care Plan.",
+      "continue-button": "Continue",
+      "back-button": "Back",
+      "submit-button": "Submit Application",
+      "exit-button": "Exit application",
+      "yes": "Yes",
+      "no": "No"
+    },
+    "modal": {
+      "header": "Close application",
+      "info": "By closing the application, you will end your session and any information entered will be cleared from your browser. Make sure you printed or save a copy of your application code.",
+      "are-you-sure": "Are you sure you want to close the application?",
+      "back-btn": "Back",
+      "close-btn": "Close application"
+    }
```

Comment:
We should review all i18n keys and see if they are being referenced. I noticed these modal keys aren't being used and can be removed.

Reply:
Fixed: 9ff413fd0e99d71234f9e44fef4e2aad98a8ef32